frontend/src/scenes/data-warehouse/settings/source/Syncs.tsx

```diff
@@ -38,7 +38,14 @@ export const Syncs = ({ id }: SyncsProps): JSX.Element => {
                 {
                     title: 'Status',
                     render: (_, job) => {
-                        return <LemonTag type={StatusTagSetting[job.status]}>{job.status}</LemonTag>
+                        const tagContent = (
+                            <LemonTag type={StatusTagSetting[job.status] || 'default'}>{job.status}</LemonTag>
+                        )
+                        return job.latest_error && job.status === 'Failed' ? (
```

Comment:
Minor nit, but I like using the enum instead of a string - makes it easier to change down the line if we ever need to: `ExtrernalDataJob.Status.FAILED` instead of `"Failed"`